doc/explanation/security-overview.rst

```diff
@@ -1,38 +1,38 @@
-.. _security-overview
+.. _subiquity-security-overview:
 
-Security overview
-=================
+Subiquity security overview
+===========================
 
-This explanation covers several security related topics for the Subiquity and
-Ubuntu-desktop-bootstrap installation ISOs.
+This explanation covers several security-related topics for the Subiquity and
+Ubuntu-desktop-bootstrap installation ISO images.
 
 
 About the installer user
 ------------------------
 
 At installation time, the default user should be considered to have root
-privileges.  The install system must be able to make arbitrary changes to the
-target system so that the install can complete successfully.  Additionally,
-there is an ``/etc/sudoers.d`` ``NOPASSWD`` entry for the default user, which
+privileges.  The installation system must be able to make arbitrary changes to the
+target system, so that the installation can complete successfully.  Additionally,
+there is an ``NOPASSWD`` entry in the :file:`/etc/sudoers.d` for the default user, which
 means that the default installer user can become root at any time with a
-``sudo`` invocation.
+:command:`sudo` invocation.
 
 
-Ubuntu-server ISO has listening by default with a random password
------------------------------------------------------------------
+Ubuntu-server ISO is listening by default with a random password
+----------------------------------------------------------------
 
-The Ubuntu Server ISO offers SSH access to the installation system, to
-facilitate installs which need to start over a minimal serial line that may not
+The Ubuntu Server ISO offers SSH access to the installation system to
+facilitate installations that need to start over a minimal serial line, and that may not
```

Comment:
This is intending to talk about two use cases
1. serial access, which may or may not be rich enough to run the installer in any capacity
2. other users who for their own reasons would prefer SSH access - richer language support is one reason

Reply:
Thanks, @dbungert. I modified the wording to reflect your note. I'm curious though: how does SSH access offer richer lang. support?

Reply:
Thank you!